app/controllers/sessions_controller.rb

```diff
@@ -14,8 +15,16 @@ def create
   end
 
   def destroy
-    log_out
+    log_out if logged_in?
     flash[:success] = t ".message_logout"
     redirect_to root_path
   end
+
+  def check_remember user
+    params[:session][:remember_me] == if Settings.sessions.value_remember
```

Comment:
cái này em check gì đây em

Reply:
Dạ cái này dùng để check xem cái params[:session][:remember_me] = 1 hay = 0 để thực hiện remember hay forget user á anh, để ở create nó phình to nên em tách riêng ra 1 method riêng ạ